test/IDE/complete_keywords.swift

```diff
@@ -401,3 +404,16 @@ func inExpr5() {
 func inExpr6() -> Int {
   return #^EXPR_6^#
 }
+
+func inSwitch(val: Int) {
+  switch val {
+  #^SWITCH_TOP^#
+  case 1:
+    foo()
+  #^SWITCH_IN_CASE^#
+  }
+// Begin completions
+// KW_CASE-DAG: Keyword[case]/None:                 case; name=case
+// KW_CASE-DAG: Keyword[default]/None:              default; name=default
```

Comment:
Bonus points (starter bug?): refuse to private `default` unless the completion's happening at the end of the switch.

Reply:
We should implement syntactic context aware keyword completion mechanism. Currently we provide `fallthrough`, `break`, et al. even in top-level.